sklearn/inspection/_plot/partial_dependence.py

```diff
@@ -539,6 +553,10 @@ def plot(self, ax=None, n_cols=3, line_kw=None, contour_kw=None):
             The maximum number of columns in the grid plot. Only active when
             `ax` is a single axes or `None`.
 
+        centered : bool, default=False
+            Center the plotted line(s). Anchor for centering is the first value
```

Comment:
We could mention that all line(s) will be anchored to the point `(x=x[0], y=0)`.

Reply:
Sounds like good idea! I'll update the doc.